beta/project-griffon/using-project-griffon/versions-view.md

```diff
@@ -0,0 +1,27 @@
+# Versions Assurance View
+
+## Overview
+
+You may use the Extensions Version view in order to quickly and easily view which AEP extensions you have installed and if they are up to date in a client connected to a Project Griffon session. 
+
+## Getting Started
+
+To use this view, complete the following steps:
+
+1. [Setup Project Griffon](../set-up-project-griffon.md)
+2. [Create](./#creating-sessions) and [connect](./#connecting-to-a-session) to a Project Griffon session
+3. In the **Home** view, select **Extension Versions**
+
+![Extension Versions](../../../.gitbook/assets/versions-extension.png)
```

Comment:
For the screenshots, I'd remove the Dev Mode and also make sure the notifications have been read (unread notifications in screenshots are a pet peeve of mine).